packages/block-editor/src/components/block-list/test/block-invalid-warning.native.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import {
+	fireEvent,
+	getBlock,
+	getEditorHtml,
+	initializeEditor,
+} from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+beforeAll( () => {
+	// Register all core blocks
+	registerCoreBlocks();
+} );
+
+afterAll( () => {
+	// Clean up registered blocks
+	getBlockTypes().forEach( ( block ) => {
+		unregisterBlockType( block.name );
+	} );
+} );
```

Comment:
If desired, there is a [`setupCoreBlocks`](https://github.com/WordPress/gutenberg/blob/774cc49d0e40138ff7a4348e426bbaed0684db18/test/native/integration-test-helpers/setup-core-blocks.js) helper that performs this logic.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/50108/commits/d16e89670eee9debd1787cce60950520253549fd